### ISSUE TITLE
[bug]: UI Validation Error

### ISSUE THREAD

**Issue by KennethLj** [invoke-ai/InvokeAI#4304]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

macOS

### GPU

mps

### What version did you experience this issue on?

3.0.2-post1

### What happened?

When pressing the Invoke button an error message is shown saying "Server Error: Validation Error". This error was mistakenly reported also at the end of another bug report #4098 . Following the instructions given to report this error I copied the object from the error message in the javascript console as following:

[Error]  error  [session]: Problem creating session 
Object

error: {detail: Array}

graph: {graph: Object}

status: 422

Object-prototyp

	error
	(anonym funktion) (index-deaa1f26.js:136)
	(anonym funktion) (index-deaa1f26.js:128:6424)
	T (index-deaa1f26.js:128:3796)
	(anonym funktion) (index-deaa1f26.js:128:2489)
	effect (index-deaa1f26.js:138:45958)
	(anonym funktion) (index-deaa1f26.js:40:105376)
	l (index-deaa1f26.js:40:84505)
	(anonym funktion) (index-deaa1f26.js:40:85469)
	Promise
	im (index-deaa1f26.js:40:85257)
	(anonym funktion) (index-deaa1f26.js:40:106385)
	(anonym funktion) (index-deaa1f26.js:40:98614)
	l (index-deaa1f26.js:40:84505)
	s (index-deaa1f26.js:40:85350)

**Comment by lstein** [comment]
You may have a cached copy of the frontend files. Try going to settings and selecting “Reset WebUI”.

**Comment by psychedelicious** [comment]
Thanks @KennethLj . Unfortunately the relevant part of the error message didn't get copied.

In particular, there is a section that shows exactly what the error is.

Here's what it should look like:
```json
{
  "graph": {
      "graph": {
          "id": "text_to_image_graph",
          "nodes": {
              "main_model_loader": {
                  "type": "main_model_loader",
                  "id": "main_model_loader",
                  "is_intermediate": true,
                  "model": {
                      "model_name": "stable-diffusion-2-1",
                      "base_model": "sd-2",
                      "model_type": "main"
                  }
              },
              "clip_skip": {
                  "type": "clip_skip",
                  "id": "clip_skip",
                  "skipped_layers": 0,
                  "is_intermediate": true
              },
              ...
          },
          "edges": [
              {
                  "source": {
                      "node_id": "main_model_loader",
                      "field": "unet"
                  },
                  "destination": {
                      "node_id": "denoise_latents",
                      "field": "unet"
                  }
              },
              {
                  "source": {
                      "node_id": "main_model_loader",
                      "field": "clip"
                  },
                  "destination": {
                      "node_id": "clip_skip",
                      "field": "clip"
                  }
              },
              ...
          ]
      }
  },
  "status": 422,
  "error": {
      "detail": [ // <---- this section will tell us exactly what is wrong
          {
              "loc": [
                  "body",
                  "nodes",
                  "denoise_latents",
                  "DenoiseLatentsInvocation",
                  "is_intermediate"
              ],
              "msg": "value could not be parsed to a boolean",
              "type": "type_error.bool"
          }
      ]
  }
}
```

**Comment by psychedelicious** [comment]
@lstein We already went through some triage on this issue, I'll let you know if it ends up being related to installer.

**Comment by lstein** [comment]
> @lstein We already went through some triage on this issue, I'll let you know if it ends up being related to installer.

You think it might be installer related? That would be weird.

**Comment by KennethLj** [comment]
Unfortunately I can't reproduce the error any more, If I do I'll have another go at pasting the necessary error message.

**Comment by psychedelicious** [comment]
I'll close this as you haven't been able to reproduce the issue. I've fixed a bug that could have caused this in the `main` branch; it will be included in the 3.1 release.
